src/Sylius/Bundle/TaxationBundle/Form/Type/TaxCalculationStrategyChoiceType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\TaxationBundle\Form\Type;
+
+use Sylius\Component\Taxation\Strategy\TaxCalculationStrategies;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Mark McKelvie <mark.mckelvie@reiss.com>
+ */
+class TaxCalculationStrategyChoiceType extends AbstractType
+{
+    protected $strategies = [
```

Comment:
It's protected because this is the same way that Sylius does it for other ChoiceTypes. When I saw a similar pattern elsewhere on Sylius I thought that to get it working I would list the strategy types here. Of course this means I would have to extend it on Reiss and we can avoid that by injecting it into the constructor instead. That's what I'm going to try now.